### DISCUSSION TITLE
Propose to add Sei assets to wormhole-token-list repo

### DISCUSSION THREAD

**Discussion by alvinkan** [wormhole-foundation/wormhole#3345] [General] [answered]
I'd like to request for Sei assets from pacific-1 (mainnet) from wormhole to this repo: 
https://github.com/wormhole-foundation/wormhole-token-list

Here is the reference for Sei assets: 
https://github.com/sei-protocol/chain-registry/blob/main/assetlist.json

**Comment by aadam-10** [comment, marked as answer]
hi @alvinkan, the token list is no longer used. you can reach out to us on Discord or other channels where we can assist with your usecase.
